osquery/tables/system/linux/processes.cpp

```diff
@@ -13,8 +13,13 @@
 
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/types.h>
```

Comment:
It shows up in `man 2 stat`, but I can try removing it and making sure it still compiles.

```
       #include <sys/types.h>
       #include <sys/stat.h>
       #include <unistd.h>

       int stat(const char *path, struct stat *buf);
       int fstat(int fd, struct stat *buf);
       int lstat(const char *path, struct stat *buf);
```